.eslintrc-webapp.json

```diff
@@ -2,21 +2,18 @@
   "root": true,
   "extends": [
     "plugin:mattermost/react",
-    "plugin:cypress/recommended",
```

Comment:
Nope, still being used. I think it's good not to include this to our [eslint-plugin-mattermost](https://github.com/mattermost/eslint-plugin-mattermost) since not all repos are using Cypress. Others are using Playwright and Detox.
But I'll defer to you for it to remain as is.

Reply:
I'd like to keep the webapp ESLint file the way it is, but if we can then override it in the Desktop App one I'd be fine removing these plugins.